models/benchmarks/himeno_model.py

```diff
@@ -35,7 +53,8 @@ def prepare_build_benchmark(self, extra_deps):
         self.build_root = os.path.join(self.benchmark_rootpath, self.name)
         prepare_cmds = [[]]
         prepare_cmds.append(['mkdir', self.build_root])
-        prepare_cmds.append(['wget', '-P', self.build_root , self.benchmark_url])
+        prepare_cmds.append(
+            ['wget', '-P', self.build_root, self.benchmark_url])
```

Comment:
spurious change? it was better the old way. :)

Reply:
AutoPep8